doc/operator-notes.md

```diff
@@ -30,3 +30,35 @@ In the second two cases, where there is a preexisting filesystem, Ignition's beh
 If `wipeFilesystem` is set to true, Ignition will always wipe any preexisting filesystem and create the desired filesystem. Note this will result in any data on the old filesystem being lost.
 
 If `wipeFilesystem` is set to false, Ignition will then attempt to reuse the existing filesystem. If the filesystem is of the correct type, has a matching label, and has a matching UUID, then Ignition will reuse the filesystem. If the label or UUID is not set in the Ignition config, they don't need to match for Ignition to reuse the filesystem. Any preexisting data will be left on the device and will be available to the installation. If the preexisting filesystem is *not* of the correct type, then Ignition will fail, and the machine will fail to boot.
+
+## selinux
+
+When using Ignition with distributions which rely on [selinux][selinux] for additional security extra care must be taken to prevent Ignition from breaking the system. Unfortunately a typical setup in a distribution is to not include selinux policies in the initramfs where Ignition runs, so any files, directories, and links created by Ignition don't receive the proper default selinux labels they should.
```

Comment:
`for additional security,`
`Unfortunately, distributions typically do not include SELinux policies in the initramfs`
s/they should//

Reply:
OK I spent a bit of time refreshing my memory on things here. One useful commit
is: https://github.com/systemd/systemd/commit/37f78db2f4a33474fc349f406b0a0a48e9c573a2

Including policy in the initramfs would mean that customization gets a lot
harder, you'd have to rebuild the initramfs after doing `sebool` etc.

The other alternative is having systemd load policy before doing the pivot, from
inside the initramfs. (I had thought it works this way, but based on the above
commit and actually looking at things it definitely doesn't, i.e. policy is
loaded after the pivot). Changing that's going to be somewhat hard for us.
Although offhand...I think Android works that way?

Another way to handle this would be to split Ignition tasks into two phases;
"partitioning things" i.e. those before the root filesystem (or `/var`) that
need to happen in the initramfs, and those in very early boot. I know a
rationale for Ignition vs Ubuntu cloud-init is how the units are processed, but
systemd already has [generators](https://www.freedesktop.org/software/systemd/man/systemd.generator.html)
which are quite close to what we want.

Reply:
The problem with moving things out of the initramfs is you might want to configure things that are needed very early in the real root. For example, adding an entry to `/etc/fstab` or writing a generator. The files stage really does also need to be in the initamfs, otherwise it's just invitings races everywhere.